packages/@vuepress/theme-default/src/client/styles/normalize.scss

```diff
@@ -14,7 +14,7 @@ body {
   font-family: var(--font-family);
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  font-size: 16px;
+  font-size: 1rem;
```

Comment:
The `normalize.scss` file is to ensure most elements to have normalized styles, instead of browser-specific styles. If changing this to `1rem`, it will depend on the browser's preset.

Reply:
But actually setting font-size to body has no effect on `rem`, if user set root font size with `32px`, VuePress still render a SUPER big heading 1.